src/components/transport_manager/test/transport_manager_impl_test.cc

```diff
@@ -633,15 +633,15 @@ TEST_F(TransportManagerImplTest, SendMessageToDevice_SendDone) {
   EXPECT_TRUE(waiter.WaitFor(1, kAsyncExpectationsTimeout));
 }
 
-TEST_F(TransportManagerImplTest, SendMessageFailed_GetHandleSendFailed) {
+TEST_F(TransportManagerImplTest, SendMessageToDevice_SendFailedEvent) {
```

Comment:
@mvorobio Please name it according to the pattern < WhatIsTested_Preconditions_Expectations >

Reply:
@AByzhynar please check https://github.com/LuxoftSDL/sdl_core/pull/18/commits/df8c18c6c1f540cb8fdd56fff93670282a59de09